tutorials/notebooks/mct_features_notebooks/pytorch/example_pytorch_mobilenet_gptq.ipynb

```diff
@@ -0,0 +1,485 @@
+    "\n",
+    "GPTQ is a gradient-based optimization process, which requires representative dataset to perform inference and compute gradients. It is possible to define a separate representative dataset in addition to the one used for the PTQ statistics collection. A possible reason to do so is, for example, to use a larger dataset in the optimization process.\n",
+    "\n",
+    "In this tutorial we use the same representative dataset for both statistics collection and GPTQ. \n",
+    "A complete pass through the representative dataset generator constitutes an epoch (batch_size x n_iter samples).\n",
+    "In this example we use the same dataloader iterator for all epochs, i.e. different images are used in different epochs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ed56f505-97ff-4acb-8ad8-ef09c53e9d57",
+   "metadata": {
+    "id": "ed56f505-97ff-4acb-8ad8-ef09c53e9d57"
+   },
+   "outputs": [],
+   "source": [
+    "batch_size = 50\n",
+    "n_iter = 10\n",
+    "\n",
+    "dataloader = DataLoader(dataset, batch_size=batch_size, shuffle=True)\n",
+    "dataloader_iter = iter(dataloader)\n",
+    "\n",
+    "def representative_dataset_gen():\n",
+    "    for _ in range(n_iter):\n",
+    "        yield [next(dataloader_iter)[0]]\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8a8b486a-ca39-45d9-8699-f7116b0414c9",
+   "metadata": {
+    "id": "8a8b486a-ca39-45d9-8699-f7116b0414c9"
+   },
+   "source": [
+    "## Model Gradient-Based Post-Training quantization using MCT\n",
+    "\n",
+    "This is the main part in which we quantize and our model.\n",
+    "\n",
+    "Next, we create a **GPTQ configuration** with possible GPTQ optimization options (such as the number of epochs for the optimization process). \n",
+    "MCT will quantize the model and start the GPTQ process to optimize the model's parameters and quantization parameters.\n",
+    "\n",
+    "In addition, we need to define a `TargetPlatformCapability` object, representing the HW specifications on which we wish to eventually deploy our quantized model."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "2edacb5b7779e4d8",
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "from model_compression_toolkit.gptq.common.gptq_constants import REG_DEFAULT\n",
+    "\n",
+    "# Specify the target platform capability (TPC)\n",
+    "tpc = mct.get_target_platform_capabilities(\"pytorch\", 'imx500', target_platform_version='v1')\n",
+    "\n",
+    "# Create a GPTQ quantization configuration and set the number of training iterations. \n",
+    "# For the sake of running faster, the hessian based weights are disabled in this tutorial.\n",
+    "gptq_config = mct.gptq.get_pytorch_gptq_config(n_epochs=50,\n",
+    "                                               use_hessian_based_weights=False,\n",
```

Comment:
Please check the following:
Set use_hessian_based_weights to True, and run it with the nightly MCT version (change the import just for the sake of this test, don't push it to the actual tutorial).
Monitor how longer is the execution, compared to a run without the Hessians.
If it is not significant (say+-10 minutes), then, set use_hessian_based_weights to True by default in the tutorial (or don't mention it at all because this is the default value anyway) and modify the comment above the code block (say that the execution might take some time)

Reply:
Also verify the difference in the results (accuracy)

Reply:
Changed. The whole run takes less than 5min as I only use 50 epochs. I'm not sure why in keras tutorial 5000 is used.